packages/Card/stories/Card.screener.stories.js

```diff
@@ -1,12 +1,12 @@
 import React from "react";
+import { getStoryName } from "storybook/storyTree";
 import Screener from "./examples/Screener";
 
 export default {
-  title: " | Card/Automation Tests",
+  title: `${getStoryName("Card")}/Backyard/Tests`,
```

Comment:
Hey buddy, why don't you use a `const` here, instead of inlining the `getStoryName()`? This is a `js` file, not a weird `mdx` file that requires you to `export` every declaration!

Reply:
Oh! Right, good call, buddy.

Reply:
I see no problem with inlining here 😄